.github/workflows/release_image.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
took the time to bump actions tags to avoid build deprecation warnings about node 12